src/main/java/org/folio/auth/authtokenmodule/apis/RouteApi.java

```diff
@@ -150,21 +163,30 @@ private void handleSignToken(RoutingContext ctx) {
       if (isDummyTokenSigningRequest(payload)) {
         logger.debug("Signing request is for a dummy token");
 
-        token = new DummyToken(tenant, payload.getJsonArray("extra_permissions"), username);
+        var dt = new DummyToken(tenant, payload.getJsonArray("extra_permissions"), username);
+        responseObject.put("token", dt.encodeAsJWT(tokenCreator));
+        endJson(ctx, 201, responseObject.encode());
       } else {
         logger.debug("Signing request is for an access token");
 
-        String userId = payload.getString("user_id");
-        token = new AccessToken(tenant, username, userId);
+        String userId = payload.getString(USER_ID);
+
+        // Generate the access token.
+        var at = new AccessToken(tenant, username, userId);
+        responseObject.put(ACCESS_TOKEN, at.encodeAsJWT(tokenCreator));
+
+        // Generate the refresh token.
```

Comment:
The comments

    // Generate the access token.

and

    // Generate the refresh token.

are not needed because the code sections are short and every Java programmer can easily get this information from the code (`new AccessToken`, `new RefreshToken`).